Compiler.cmake

```diff
@@ -11,6 +11,7 @@
 # Input Variables
 # * COMMON_MINIMUM_GCC_VERSION check for a minimum gcc version, default 4.4
 # * COMMON_USE_CXX03 When set, do not enable C++11 language features
+# * COMMON_DISABLE When set, do not set any flags
```

Comment:
@mgeplf : This is one of the many new things in CMake 3. Unfortunately I'm not sure we can abandon CMake 2.8 yet.

F2F sounds good - Catch us between Lunch and the 2pm presentation? Or let's have lunch together>